cmd/cli/proxy_get.go

```diff
@@ -1,10 +1,9 @@
 package main
 
 import (
-	"context"
-	"fmt"
+	"bufio"
+	"github.com/openservicemesh/osm/pkg/cli"
```

Comment:
Push this to the bottom (openservicemesh/osm imports should be in their own grop, at the bottom, last group, separated by the other imports with a new line)